### DISCUSSION TITLE
Sending HTML as mail body

### DISCUSSION THREAD

**Discussion by m1abdullahh** [googleapis/googleapis#877] [Q&A] [answered]
Hey, everyone.
I was trying to send some automated mails using googleapis, I was wondering if there was a way to send rendered html as the mail body the way we can in something like node-mailer.
Any help would be appereciated.

**Comment by xionams** [comment, marked as answer]
You can.
read more about it [here](https://developers.google.com/gmail/api/guides/sending#python)
